README.md

```diff
@@ -1 +1,10 @@
-# Fitness-Accountability
\ No newline at end of file
+# Fitness-Accountability
+
+- Name: Have this alphabetic
+- Why
+- Weekly: What is your schedule
+- Goal
+
+| NAME | WHY | WEEKLY | GOAL |
+| :--- | :--- | :--- | :--- |
+| [Eddie Jaoude](http://github.com/eddiejaoude) | To have a break from my desk | Spin x3, Cycle x2, Gym x2 | - |
```

Comment:
We could use HTTPS links -
```suggestion
| [Eddie Jaoude](https://github.com/eddiejaoude) | To have a break from my desk | Spin x3, Cycle x2, Gym x2 | - |
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix